packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -936,22 +845,28 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 }
 
 /**
- * For a blobId, returns its path in GC's graph. The node path is of the format `/<BlobManager.basePath>/<blobId>`.
+ * For a blobId, returns its path in GC's graph. The node path is of the format `/<blobManagerBasePath>/<blobId>`.
  * This path must match the path of the blob handle returned by the createBlob API because blobs are marked
  * referenced by storing these handles in a referenced DDS.
  */
-function getGCNodePathFromBlobId(blobId: string) {
-	return `/${BlobManager.basePath}/${blobId}`;
-}
+const getGCNodePathFromBlobId = (blobId: string) => `/${blobManagerBasePath}/${blobId}`;
```

Comment:
Are arrow functions better for some reason in terms of bundle size or something? Or you just did it since it directly returns a value.